server/data/strapiApiClient.test.ts

```diff
@@ -289,7 +289,7 @@ describe('strapiApiClient', () => {
   describe('postGithubRepoRequest', () => {
     it('should insert a single form request', async () => {
       const response = {
-        data: { documentId: 'documentid1', name: 'GIthub repo request form' },
+        data: { documentId: 'documentid1', name: 'GIthub repo request form', request_type: 'Add' },
```

Comment:
can we use normal naming conventions for enums? `UNDERSCORE_UPPERCASE`, so:  `ADD`